remote_dumps/README.md

```diff
@@ -3,12 +3,13 @@ This set of scripts allows to download, parse and aggregate BGP announcement dum
 ### Pre-reqs
 
 ``./setup.sh``
+``pip install bgpdumpy``
```

Comment:
Fine to do this as part of the setup script? Maybe we should also just have a requirements.txt anyway.

Reply:
Perfect, gonna do it